### ISSUE TITLE
PRACK support

### ISSUE THREAD

**Issue by getsok** [sipsorcery/sipsorcery#80]
Hi! Do you plan to add support PRACK?

**Comment by sipsorcery** [comment]
Yes should be able to get it done next week.

**Comment by sipsorcery** [comment]
For context original request came from https://stackoverflow.com/questions/58117975/prack-request-using-sipsorcery.

> how to send a message PRACK?
I give so:

````
m_uac.CallRinging += (ISIPClientUserAgent uac, SIPResponse sipResponse) =>
{

if ((sipResponse.StatusCode == (int)SIPResponseStatusCodesEnum.SessionProgress ||
sipResponse.StatusCode == (int)SIPResponseStatusCodesEnum.Ringing) && 
sipResponse.Header.Require == "100rel")
    {
      m_uac.SendRequest(sipResponse, SIPMethodsEnum.PRACK, null, $"RAck: {sipResponse.Header.RSeq} {sipResponse.Header.CSeq} {sipResponse.Header.CSeqMethod}")
    }

 }
````

> get errors: Exception CreateNonInviteTransaction. An attempt was made to add a duplicate SIP transaction. Exception SIPNonInviteClientUserAgent SendRequest to sip:test@sip.srv_1.com. An attempt was made to add a duplicate SIP tra Exception UACInviteTransaction_TransactionInformationResponseReceived. An attempt was made to add a duplicate SIP transaction.

**Comment by sipsorcery** [comment]
PRACK support is now done and available on `master` branch and in the [v3.3.0 nuget package](https://www.nuget.org/packages/SIPSorcery/3.3.0).

Nothing special is required to enable it. If placing a call and the server returns `100rel` in the `Supported` header then PRACK's will automatically be used. Same thing if receiving a call and the client sets the `Supported` header.
